scipy/signal/signaltools.py

```diff
@@ -2862,17 +2866,25 @@ def resample(x, num, t=None, axis=0, window=None):
     >>> plt.legend(['data', 'resampled'], loc='best')
     >>> plt.show()
     """
+
+    if domain not in ('time', 'freq'):
+        raise ValueError("Acceptable domain flags are 'time' or"
+                         " 'freq', not domain={}".format(domain))
+
     x = np.asarray(x)
     Nx = x.shape[axis]
 
     # Check if we can use faster real FFT
     real_input = np.isrealobj(x)
 
-    # Forward transform
-    if real_input:
-        X = sp_fft.rfft(x, axis=axis)
-    else:  # Full complex FFT
-        X = sp_fft.fft(x, axis=axis)
+    if domain == 'time':
+        # Forward transform
+        if real_input:
+            X = sp_fft.rfft(x, axis=axis)
+        else:  # Full complex FFT
+            X = sp_fft.fft(x, axis=axis)
+    elif domain == 'freq':
```

Comment:
Now it is just `else`. You can write 

```python
    else:  # domain == 'freq'
```
Note double whitespace before `#` for PEP8